ecommerce/models.py

```diff
@@ -38,10 +38,42 @@ def __str__(self):
 class Product(models.Model):
```

Comment:
Add a v BooleanField `active` in it.

Reply:
actie_product = models.BooleanField(default=True)
is it right way??